git-ftp

```diff
@@ -789,10 +789,12 @@ submodule_catchup() {
 	set_submodule_args
 	url="$(git config git-ftp.url)"
 	print_info "Submodules are $GIT_SUBMODULES"
-	for submodule in "${GIT_SUBMODULES[@]}"
+	for submodule in ${GIT_SUBMODULES[@]}
```

Comment:
This will fail as soon as a submodule has a space in the path. I think the real bug is in line 237 when the variable is assigned as one string instead of an array.